operating/manage-compliance/compliance-feature-overview.adoc

```diff
@@ -29,6 +29,11 @@ Formerly known as _Compliance 2.0_, summarizes the compliance information in a s
 For more information, see xref:../../operating/manage-compliance/using-openshift-compliance.adoc#using-openshift-compliance[Using OpenShift compliance].
 ====
 
+[NOTE]
+====
+To access the Compliance menu and API, a user must have read or write permissions for the `Compliance` resource and read permissions for the `Cluster` resource. Additionally, to use email notifiers in the scan schedule configuration, the user requires read permissions for the `Integration` resource.
```

Comment:
[nit] Maybe use `at least read permissions` instead of `read or write` for both permissions.